components/molecule/dropdownList/src/index.js

```diff
@@ -78,6 +78,8 @@ const MoleculeDropdownList = ({
     ev.stopPropagation()
   }
 
+  if (!visible) return null
```

Comment:
This should be behind a prop as sometimes, for SEO reasons, some verticals could want to still render the content.